internal/httputil/certutil.go

```diff
@@ -34,10 +32,10 @@ func NewCertPool(caDir string, log logr.Logger) (*x509.CertPool, error) {
 			return nil, err
 		}
 		if fi.IsDir() {
-			log.Info("skip directory", "name", e.Name())
+			log.V(defaultLogLevel).V(1).Info("skip directory", "name", e.Name())
 			continue
 		}
-		log.Info("load certificate", "name", e.Name())
+		log.V(defaultLogLevel).V(1).Info("load certificate", "name", e.Name())
```

Comment:
Just checking that you are intending to use `V(defaultLogLevel).V(1)` here (which is the same as `V(defaultLogLevel+1)`)? If so, maybe use the latter construction?

Reply:
Yes, that was the intent (do to defaultLogLevel+1)